### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: po/common/fr.po [hunk 1]
```diff
@@ -4324,7 +4324,7 @@ msgstr "Code-barres invalide"
 # Either 'ltr' for left to right languages like English or 'rtl' for right to left languages like Arabic
 msgctxt "text_direction"
 msgid "ltr"
-msgstr ""
+msgstr "de gauche à droite"
```
Comment (svensven) [inline po/common/fr.po:4327]:
```suggestion
msgstr ""
```

Comment (teolemon) [reply po/common/fr.po:4327]: arg. we need to clean it in crowdin, otherwise it will happen again

Comment (teolemon) [reply po/common/fr.po:4327]: https://crowdin.com/translate/openfoodfacts/1378/en-fr?filter=basic&value=0#q=ltr
fixed

Comment (svensven) [reply po/common/fr.po:4327]: at least the test stopped it breaking the site this time :)

File: po/common/fr.po [hunk 2]
```diff
@@ -4380,15 +4380,15 @@ msgstr "Texte ou phrases indiquant l'origine du produit et / ou de ses ingrédie
...
+msgstr "Note Nutri-Score de A à E affichée sur l'étiquette produit"
 
 msgctxt "nutriscore_grade_producer_import_note"
 msgid "Open Food Facts computes the Nutri-Score grade based on the information provided (nutrition facts and category). If the grade we compute is different from the grade you provide, you will get a private notification on the producers platform so that the difference can be resolved."
 msgstr "Open Food Facts calcule le Nutri-Score selon les informations fournies (catégorie et informations nutritionnelles). Si la note que nous calculons est différent de la note que vous fournissez, vous recevrez une notification privée sur votre plateforme producteur afin que nous puissions résoudre cette disparité."
 
 msgctxt "nutriscore_score_producer_note"
 msgid "Nutri-Score score (numeric value from which the A to E grade is derived)"
-msgstr "Score Nutri-Score (valeur numérique à partir de laquelle la note de A à E est attribuée)"
+msgstr "Note Nutri-Score (valeur numérique de laquelle est dérivée le rang A à E)"
```
Comment (stephanegigandet) [inline po/common/fr.po:4391]: I can't find those strings in crowdin, I'd like to change some of the French translations...

https://translate.openfoodfacts.org/translate/openfoodfacts/440/en-fr?filter=basic&value=0#q=nutriscore_score_producer_note

**Review by stephanegigandet — APPROVED** [review]

File: po/common/fr.po (hunk 2 above)
Comment (teolemon) [reply po/common/fr.po:4391]: "X-Crowdin-File-ID: 1378\n"
https://translate.openfoodfacts.org/translate/openfoodfacts/1378/en-fr?filter=basic&value=0#q=nutriscore_score_producer_note

**Comment by teolemon** [comment]
RROR - fr:algues-euchema-transformees already is a synonym of fr:e407a for entry en:e407a - fr:algues-euchema-transformees cannot be mapped to entry en:processed-euchema-algae
ERROR - nl:verwerkt-eucheuma-wier already is a synonym of nl:e407a for entry en:e407a - nl:verwerkt-eucheuma-wier cannot be mapped to entry en:processed-euchema-algae
ERROR - ar:محار already is a synonym of ar:رخويات for entry en:mollusc - ar:محار cannot be mapped to entry en:oyster
[ingredients] child process exited with code 0
Done!

**Comment by svensven** [comment]
poking travis...

File: html/donate/he.html [hunk 3]
```diff
@@ -51,7 +51,7 @@
 
     <link
       rel="stylesheet"
-      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
+      href="https://static.openfoodfacts.org/css/dist/app-ltr.css"
```
Comment (teolemon) [inline html/donate/he.html:54]:
```suggestion
      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
```

File: html/donate/ar.html [hunk 4]
```diff
@@ -51,7 +51,7 @@
 
     <link
       rel="stylesheet"
-      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
+      href="https://static.openfoodfacts.org/css/dist/app-ltr.css"
```
Comment (teolemon) [inline html/donate/ar.html:54]:
```suggestion
      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
```

File: html/donate/fa.html [hunk 5]
```diff
@@ -51,7 +51,7 @@
 
     <link
       rel="stylesheet"
-      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
+      href="https://static.openfoodfacts.org/css/dist/app-ltr.css"
```
Comment (teolemon) [inline html/donate/fa.html:54]:
```suggestion
      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
```

File: html/donate/fr.html [hunk 6]
```diff
@@ -191,12 +191,12 @@ <h3>Aidez-nous à financer le budget Open Food Facts 2020 !</h3>
...
           ></script>
           <iframe
-            title="Donate via Donorbox"
+            title="Faire un don via Donorbox"
             allowpaymentrequest=""
             height="900px"
             name="donorbox"
             seamless="seamless"
-            src="https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget"
+            src="https://donorbox.org/embed/your-donation-to-open-food-facts"
```
Comment (teolemon) [inline html/donate/fr.html:199]:
```suggestion
            src="https://www.helloasso.com/associations/open-food-facts/formulaires/1/widget"
```

File: html/donate/fr.html [hunk 7]
```diff
@@ -191,12 +191,12 @@ <h3>Aidez-nous à financer le budget Open Food Facts 2020 !</h3>
             paypalExpress="true"
           ></script>
           <iframe
-            title="Donate via Donorbox"
+            title="Faire un don via Donorbox"
```
Comment (teolemon) [inline html/donate/fr.html:194]:
```suggestion
            title="Faire un don via Hello Asso"
```

File: html/donate/hi.html [hunk 8]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
 
-    <title>Open Food Facts को दान करें</title>
+    <title>Donate to Open Food Facts</title>
```
Comment (teolemon) [inline html/donate/hi.html:6]:
```suggestion
    <title>Open Food Facts को दान करें</title>
```

File: lang/br/texts/open-food-hunt.html [hunk 9]
```diff
@@ -11,7 +11,7 @@ <h1>Open Food Hunt</h1>
 </p>
 
 <img id="og_image" src="https://world.openfoodfacts.org/images/misc/open-food-hunt-2015.652x446.png" alt="Open Food Hunt" width="652" height="446" />
-<br/><em>Skeudennadur gant François Lecomte</em><br/>
+<br/><em>Illustration by François Lecomte</em><br/>
```
Comment (teolemon) [inline lang/br/texts/open-food-hunt.html:14]:
```suggestion
<br/><em>Skeudennadur gant François Lecomte</em><br/>
```

File: lang/bn/texts/index.html [hunk 10]
```diff
@@ -7,15 +7,15 @@ <h1>Open Food Facts</h1>
...
+<p>আপনি এটি আরও ভাল খাবারের পছন্দ করতে পারেন এবং এটি ওপেন ডেটা, যে কেউ যে কোনও উদ্দেশ্যেই এটি পুনরায় ব্যবহার করতে পারেন ।</p>
 
 <p>&rarr; <a href="/discover">Open Food Facts সম্পর্কে আরও জানুন </a></p>
 
 	</div>
 
 	<div class="medium-12 large-6 columns">
 
-<h2>সহযগীতা</h2>
+<h2>Contribute</h2>
```
Comment (teolemon) [inline lang/bn/texts/index.html:18]:
```suggestion
<h2>সহযগীতা</h2>
```

File: lang/el/texts/nutriscore.html [hunk 11]
```diff
@@ -58,9 +58,9 @@ <h3>Install the Open Food Facts mobile app</h3>
...
 
-		<a href="https://apps.apple.com/gr/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_GR.svg" alt="Available on the App Store" width="120" height="40"></a>
+		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_GR.svg" alt="Available on the App Store" width="120" height="40"></a>
 		<a href="https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&hl=el"><img src="https://world.openfoodfacts.org/images/misc/google-play-badge-svg-master/img/el_get.svg" alt="Available on Google Play" width="102" height="40"></a>
-		<a href="https://www.microsoft.com/el-gr/p/openfoodfacts/9nblggh0dkqr"><img src="https://world.openfoodfacts.org/images/misc/microsoft/Greek.svg" alt="Windows Phone Store" width="109" height="40"></a>
+		<a href="https://www.microsoft.com/en-us/p/openfoodfacts/9nblggh0dkqr"><img src="https://world.openfoodfacts.org/images/misc/microsoft/Greek.svg" alt="Windows Phone Store" width="109" height="40"></a>
```
Comment (teolemon) [inline lang/el/texts/nutriscore.html:63]:
```suggestion
		<a href="https://www.microsoft.com/el-gr/p/openfoodfacts/9nblggh0dkqr"><img src="https://world.openfoodfacts.org/images/misc/microsoft/Greek.svg" alt="Windows Phone Store" width="109" height="40"></a>
```

File: lang/el/texts/nutriscore.html [hunk 12]
```diff
@@ -58,9 +58,9 @@ <h3>Install the Open Food Facts mobile app</h3>
 
 <p>To scan food products, get their Nutri-Score nutritional grade, the NOVA group for ultra-processed foods, allergens alerts and to decypher food additives, install the free Open Food Facts app!</p>
 
-		<a href="https://apps.apple.com/gr/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_GR.svg" alt="Available on the App Store" width="120" height="40"></a>
+		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_GR.svg" alt="Available on the App Store" width="120" height="40"></a>
```
Comment (teolemon) [inline lang/el/texts/nutriscore.html:61]:
```suggestion
		<a href="https://apps.apple.com/gr/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_GR.svg" alt="Available on the App Store" width="120" height="40"></a>
```

File: html/donate/ur.html [hunk 13]
```diff
@@ -51,7 +51,7 @@
 
     <link
       rel="stylesheet"
-      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
+      href="https://static.openfoodfacts.org/css/dist/app-ltr.css"
```
Comment (teolemon) [inline html/donate/ur.html:54]:
```suggestion
      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
```

File: html/donate/yi.html [hunk 14]
```diff
@@ -51,7 +51,7 @@
 
     <link
       rel="stylesheet"
-      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
+      href="https://static.openfoodfacts.org/css/dist/app-ltr.css"
```
Comment (teolemon) [inline html/donate/yi.html:54]:
```suggestion
      href="https://static.openfoodfacts.org/css/dist/app-rtl.css"
```

File: lang/ar/texts/mobile-app-open-food-facts.html [hunk 15]
```diff
@@ -2,7 +2,7 @@ <h1>Install the Open Food Facts mobile app</h1>
 
 <p>To scan food products, get their Nutri-Score, their NOVA group to know if it's an ultra-processed food, and to decypher food additives, install the free Open Food Facts app!</p>
 
-		<a href="https://apps.apple.com/ar/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40" /></a>
+		<a href="https://apps.apple.com/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40" /></a>
```
Comment (teolemon) [inline lang/ar/texts/mobile-app-open-food-facts.html:5]:
```suggestion
		<a href="https://apps.apple.com/ar/app/open-food-facts/id588797948"><img src="https://static.openfoodfacts.org/images/misc/appstore/black/appstore_AR.svg" alt="Available on the App Store" width="120" height="40" /></a>
```

**Pull request closed without merging**